internal/server/kong/ws/config/configuration.go

```diff
@@ -6,8 +6,11 @@ import (
 	"context"
```

Comment:
General comment: Are we sure we want to track at a seconds granularity? Isn't that too large?

Reply:
I agree - I am accustomed to ms as the base unit. However, @tjasko pointed out that Prometheus convention is to use seconds as the base unit: https://github.com/Kong/koko/pull/485#discussion_r1003676099